gulpfile.js

```diff
@@ -0,0 +1,157 @@
+const { series, src, dest } = require('gulp')
+const { spawn } = require('child_process')
+const path = require('path')
+const PATHS = {
+    SERVER: path.resolve(__dirname, 'src', 'server'),
+    SERVER_ENTRY_POINT: path.resolve(__dirname, 'src', 'server', 'index.js'),
+    WEBPACK: path.resolve(__dirname, 'src', 'server', 'node_modules', '.bin', 'webpack'),
+    WEBPACK_PROD_CONFIG: path.resolve(__dirname, 'src', 'server', 'prod.webpack.config.js'),
+    WEBPACK_DEV_CONFIG: path.resolve(__dirname, 'src', 'server', 'dev.webpack.config.js'),
+    FRONTEND: path.resolve(__dirname, 'src', 'server', 'frontEnd'),
+    JEST_CONFIG: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'package.json'),
+    WEBPACK_GENERATED_BUNDLE: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'src', 'theme', 'index.html'),
+    HTTP_SERVER_PUBLIC_DIRECTORY: path.resolve(__dirname, 'src', 'server', 'public'),
+    ESLINT: path.resolve(__dirname, 'src', 'server', 'node_modules', 'eslint', 'bin', 'eslint.js'),
+    ESLINT_IGNORE: path.resolve(__dirname, 'src', 'server', '.eslintignore'),
+    JSDOC: path.resolve(__dirname, 'src', 'server', 'node_modules', 'jsdoc', 'jsdoc'),
+    WEBSOCKET_ROUTING_DIRECTORY: path.resolve(__dirname, 'src', 'server', 'ws-routes'),
+    NYC: path.resolve(__dirname, 'src', 'server', 'node_modules', 'nyc', 'bin', 'nyc'),
+    SERVER_COVERAGE_REPORT_DIRECTORY: path.resolve(__dirname, 'src', 'server', 'coverage'),
+    FRONTEND_COVERAGE_REPORT_DIRECTORY: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'coverage'),
+    TEST_CYPRESS: path.resolve(__dirname, 'src', 'server', 'runCypress.js'),
+    FRONTEND_ESLINT: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'node_modules', 'eslint', 'bin', 'eslint.js'),
+    FRONTEND_ESLINT_IGNORE: path.resolve(__dirname, 'src', 'server', 'frontEnd', '.eslintignore'),
+    JS_DOCS_OUTPUT_DIRECTORY: path.resolve(__dirname, 'src', 'server', 'out'),
+    FRONTEND_JEST: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'node_modules', '.bin', 'jest'),
+    FRONTEND_TEST_COVERAGE: path.resolve(__dirname, 'src', 'server', 'frontEnd', 'coverage.lcov'),
+    CONFIG_GENERATOR_SERVICE: './configFileService.js',
```

Comment:
Fix this line

Reply:
This must stay as it is because of gulpfile and configFileService are in the same directory and in ```require``` must have relative path.